### PR TITLE
Members db changes & dependencies

### PR DESCRIPTION
**Pull request by allouis**
This PR adds the dependencies required by members.

This PR adds the necessary changes to ghost database for members to work.

### CODE REVIEW

File: core/server/data/schema/schema.js [hunk 1]
```diff
@@ -368,5 +368,15 @@ module.exports = {
         mobiledoc: {type: 'text', maxlength: 1000000000, fieldtype: 'long', nullable: true},
         created_at_ts: {type: 'bigInteger', nullable: false},
         created_at: {type: 'dateTime', nullable: false}
+    },
+    members: {
+        id: {type: 'string', maxlength: 24, nullable: false, primary: true},
+        email: {type: 'string', maxlength: 191, nullable: false, unique: true, validations: {isEmail: true}},
+        name: {type: 'string', maxlength: 191, nullable: false},
+        password: {type: 'string', maxlength: 191, nullable: true},
```
Comment (allouis) [inline core/server/data/schema/schema.js]: This column is nullable to support third party auth in future. e.g. sign in with google

File: core/server/data/schema/default-settings.json [hunk 2]
```diff
@@ -118,5 +118,16 @@
         "public_hash": {
             "defaultValue": null
         }
+    },
+    "members": {
+        "members_public_key": {
```
Comment (kirrg001) [inline core/server/data/schema/default-settings.json]: Tiny design detail:

This breaks the pattern of the settings key names a bit. We don't repeat the name of the namespace in the key itself.

e.g. `blog.active_timezone` vs `members.members_public_key`

File: core/server/data/schema/schema.js (hunk 1 above)
Comment (kirrg001) [reply core/server/data/schema/schema.js]: Is there a reason that the pwd length is 191? 🤔

File: core/server/data/schema/default-settings.json (hunk 2 above)
Comment (allouis) [reply core/server/data/schema/default-settings.json]: The problem with this is that settings are actually flat, and the nesting is just for us humans - so we would have a conflict with the `session_secret` setting. And then if we were to ever have a need for a public_key or private_key setting in the future we would have to namespace it :/ 

As an alpha product i didn't think we should take up valuable non-namespaced settings names!

File: core/server/data/schema/schema.js (hunk 1 above)
Comment (allouis) [reply core/server/data/schema/schema.js]: It was the most suitable string size that we use for columns - as I didn't think we were supposed to deviate from them.  I can set it to the exact size of the bcrypt output if that's preferred?

File: core/server/data/schema/default-settings.json (hunk 2 above)
Comment (kirrg001) [reply core/server/data/schema/default-settings.json]: The settings service should probably be changed to `settingsCache.get('core.settings_secret')` 🤣 
Okay let's leave it like this for now 👍

Comment (allouis) [reply core/server/data/schema/default-settings.json]: :D agreed!

File: core/server/data/schema/schema.js (hunk 1 above)
Comment (allouis) [reply core/server/data/schema/schema.js]: Yeah totally - I was just going from this: https://github.com/TryGhost/Ghost/blob/master/core/server/data/schema/schema.js#L1-L9

I would much prefer explicit sizes - but we haven't done that recently for things like tokens. Will update this tomorrow tho

**Pull request merged**